jsf/jsf-widgets/src/java/org/sakaiproject/jsf/renderer/ViewRenderer.java

```diff
@@ -85,7 +100,7 @@ public void encodeBegin(FacesContext context, UIComponent component) throws IOEx
             }
 
 			writer.write("<!DOCTYPE html>\n");
-			writer.write("<html lang=\"" + lang + "\" dir=\"" + LocaleUtil.getOrientation(locale) + "\">\n");
+			writer.write("<html lang=\"" + lang + "\" dir=\"" + LocaleUtil.getOrientation(locale) + "\" class=\"Mrphs-html "+userTheme+"\">\n");
```

Comment:
If you add a condition to the `userTheme` variable above, we'll have to deal with the case of when that variable is not set.

Reply:
So @fostersdesign would you rather have it not have any class or have sakaiUserTheme-notSet (which I took from your code in portal)?

Reply:
I am going to merge this so progress can be made.   If @profmikegreene or @fostersdesign feel the "default" / "no pref available" needs to be different - please change it all three places (Velocuty, JSF, and Portal).